.eslintrc

```diff
@@ -222,7 +224,7 @@
     "es2020": true
   },
   "globals": {
-    "__webpack_public_path__": true,
+    "__webpack_public_path__": "writable",
```

Comment:
What does this change do and why is it needed?

Reply:
Just some cleanup along the way. Using booleans is deprecated.